### ISSUE TITLE
mobile browser

### ISSUE THREAD

**Issue by holder66** [Tmw/linreg#1]
What a neat project, Tiemen! Thank you! As an experiment, I'm running it on a raspberry pi zero w. Works great! However, when I use a mobile browser (safari on an ipad or iphone) it appears that finger taps are not picked up as events. As a newbie, can you suggest how this can be made to work on mobile devices? Thank you for any help!

**Comment by Tmw** [comment]
Thank you @holder66! You are right! I tested it on iOS Safari and indeed it seems that touch events on things other than `<button>`s are not registered for some reason.. I will do a deep dive later because this made me curious if this might be something to improve within LiveView.

But for now; maybe it might be worth making a [custom hook](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-js-interop-and-client-controlled-dom) and catch and send the event _manually_? I must say I haven't played with this feature myself so I'm not 100% sure how to go about this.

If you do find a way to make it work, let me know! :D

**Comment by Tmw** [comment]
Hey! @holder66 I took some time to figure this one out and apparently it's an iOS Safari thing where click events don't bubble up properly for elements that are non click by default: https://stackoverflow.com/questions/18524177/strange-click-event-bubble-on-iphone-safari-that-it-stop-before-bubbling-to-docu.

I fixed it in the current master by adding a `style="cursor: pointer"` attribute and assigning an empty `onclick="void(0);"` handler to the SVG element. Got it working on my end :)
